msg/msg.go

```diff
@@ -1,6 +1,6 @@
 package msg
 
-// a returns the message.
+// aaaa returns the message.
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported function GetMessage should be of the form "GetMessage ..."